### PR TITLE
Remove broken icon

### PR DESCRIPTION
**Pull request by asklar**
1. Removes broken icon in resources page:
Before:
image
After:
image

2. Prevents unnatural stretch of xbox img in showcase
Before:
image

After:
image

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows-samples/pull/601)

### CODE REVIEW

**Comment by chrisglein** [comment]
> Removes broken icon in resources page

Screenshot of before/after?

File: website/pages/en/resources-showcase.js [hunk 1]
```diff
@@ -27,13 +27,16 @@ function getBackgroundForCategory(category) {
...
+  return  <div
+  style={{
+    maxHeight: isSmall ? 40 : 600,
+    minHeight: isSmall ? 40 : 320,
+    minWidth: isSmall ? 40 : 320,
+    verticalAlign: 'baseline',
+  }}
+  ><img src={img}
+ key={key} />
+    </div>;
```
Comment (chrisglein) [inline website/pages/en/resources-showcase.js:39]: At this point you could probably scrap `img` and just set `background-image` on that `div`. Probably.

Comment (asklar) [reply website/pages/en/resources-showcase.js:39]: I don't think that does the same thing. Here's what I get if I replace img with ```backgroundImg: `url(${img})` ```

image

**Comment by asklar** [comment]
> > Removes broken icon in resources page
> 
> Screenshot of before/after?

added to description

**Review by rectified95 — APPROVED** [review]
Approving. What was the faulty icon? Do we want to replace it with something similar?

**Pull request merged**